packages/mint-components/CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.5.1] - 2022-02-28
+
+### Added
+
+- Checkbox and dropdown components added for use in \<sqm-portal-register>
+  New components:
+  - \<sqm-checkbox-field>
```

Comment:
Brief component descriptions are needed, see previous change log updates for examples